corehq/apps/geospatial/tests/test_views.py

```diff
@@ -162,6 +162,45 @@ def test_config_update(self):
         self.assertEqual(config.target_group_count, 10)
         self.assertEqual(config.selected_disbursement_algorithm, GeoConfig.RADIAL_ALGORITHM)
 
+    @flag_enabled('GEOSPATIAL')
+    def test_config_update__road_network_algorithm_ff_disabled(self):
```

Comment:
the name says "ff_disabled", I assume it was when we added a new feature flag but that is now removed?

Reply:
Ah, sharp eyes! I'll update